### ISSUE TITLE
download fails because repository changed atom/atom-shell to atom/electon

### ISSUE THREAD

**Issue by karad** [atom/grunt-download-atom-shell#22]
download fails because repository changed atom/atom-shell to atom/electon, maybe.

```
$ grunt download-atom-shell                                                             
Running "download-atom-shell" task
>> Cannot find atom-shell v0.23.0 from GitHub [Error: Not Found]
Warning: Task "download-atom-shell" failed. Use --force to continue.
```

**Comment by ducky427** [comment]
I think [L129](https://github.com/atom/grunt-download-atom-shell/blob/master/tasks/download-atom-shell-task.coffee#L129) and [L190](https://github.com/atom/grunt-download-atom-shell/blob/master/tasks/download-atom-shell-task.coffee#L190) of [tasks/download-atom-shell-task.coffee](https://github.com/atom/grunt-download-atom-shell/blob/master/tasks/download-atom-shell-task.coffee) need to be updated to reflect the new name.

**Comment by thomasjo** [comment]
This is being handled in #23. Thanks for the helpful report :bow:

**Comment by simtel12** [comment]
Another solution in #24, in a way that allows old builds of atom-shell to be used as well.

**Comment by kevinsawicki** [comment]
Fixed by #23
